from. _Error import Error


class BotPresenteError(Error):
    pass
